Modules/Devices/src/Devices.Infrastructure/OpenIddict/CustomOpenIddictEntityFrameworkCoreApplication.cs

```diff
@@ -1,8 +1,8 @@
 ﻿using OpenIddict.EntityFrameworkCore.Models;
 
-namespace Backbone.Modules.Devices.Domain.OpenIddict;
+namespace Backbone.Modules.Devices.Infrastructure.OpenIddict;
 public class CustomOpenIddictEntityFrameworkCoreApplication : OpenIddictEntityFrameworkCoreApplication<string, CustomOpenIddictEntityFrameworkCoreAuthorization, CustomOpenIddictEntityFrameworkCoreToken>
 {
-    public string TierId { get; set; }
+    public string DefaultTier { get; set; }
```

Comment:
```suggestion
    public TierId DefaultTier { get; set; }
```

You should regenerate the migrations after this. This should change the datatype from varchar to char.